mmdet/apis/train.py

```diff
@@ -68,7 +68,7 @@ def set_random_seed(seed, deterministic=False):
         torch.backends.cudnn.benchmark = False
 
 
-def train_detector(model,
+def train_model(model,
```

Comment:
Please keep the `train_detector` function and raise a deprecated warning.
Also, remember to change the related code in the tools.

Reply:
Thank you for telling me this!